src/trans/ColorMap.ts

```diff
@@ -166,7 +178,7 @@ export default class ColorMap extends Component {
                 this.main.getRctx().getGl().deleteTexture(tex);
             });
         }
-        this.colorMaps = map(this.opts.maps, (colorMap) => this._buildColorMap(colorMap));
+        this.colorMaps = map(filter(this.opts.maps, "enabled"), (colorMap) => this._buildColorMap(colorMap.colors));
```

Comment:
Can we make this backward compatible?? Will it work if we change type of `maps` in `IColorMapOpts` to `Array<IColorMapMap | IColorMapItem[]>` and then change this filtering to be something like `filter(this.opts.maps, (cm) => isColorMapMap(cm) ? cm.enabled : true)`? If it works then we can avoid updates in all the test and sample files and also maintain backward compatibility.

Reply:
Possible, but is backward compatablity (for whom?) worth the added complexity? It might be that you simply value compatablity higher than I do. Personally, I would prefer keeping the number of possible file formats at a minimum. There's going to be a "legacy" format (.avs files) and a new "webvs" format anyway, because parameter _values_ in AVS are sometimes impractical as is. So I would opt for changing the format now, and increasing the major version, if you want to follow semver; instead of supporting a component format that noone(afaik?) but us uses.

I'll gladly change anything else that needs changing. This PR should already change all the examples using ColorMap.

Reply:
I'm not really trying to persuade you, just pointing out the benefit reduced complexity. I'm fine with a no :)

Reply:
That's a fair point. I don't think bumping major version number is an issue. Let's go with changing the formats.